server.go

```diff
@@ -62,6 +62,7 @@ func (s *server) Start() (bool, error) {
 	}, bufSize)
 
 	<-s.persister.ConfigChangedWatcher()
+	s.createBucketContainer()
```

Comment:
We could create the bucket container before that - not a huge deal either way though.